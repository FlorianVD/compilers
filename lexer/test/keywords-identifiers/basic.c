return foo

if else while for

foo_bar foo123
