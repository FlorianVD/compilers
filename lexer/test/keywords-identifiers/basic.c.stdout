1:1 -> 1:7          return              RETURN              
1:8 -> 1:11         foo                 IDENTIFIER          
3:1 -> 3:3          if                  IF                  
3:4 -> 3:8          else                ELSE                
3:9 -> 3:14         while               WHILE               
3:15 -> 3:18        for                 FOR                 
5:1 -> 5:8          foo_bar             IDENTIFIER          
5:9 -> 5:15         foo123              IDENTIFIER          
